authentication/src/main/java/io/camunda/authentication/CamundaUserDetailsService.java

```diff
@@ -53,11 +54,15 @@ public UserDetails loadUserByUsername(final String username) throws UsernameNotF
 
     final Long userKey = storedUser.userKey();
 
-    final var authorizedApplications =
-        authorizationServices.getAuthorizedApplications(Set.of(storedUser.username()));
-
     final var roles = roleServices.findAll(RoleQuery.of(q -> q.filter(f -> f.memberKey(userKey))));
 
+    final var authorizedApplications =
```

Comment:
❓ groups are not supported here yet, thus not passed? I wonder whether we should extend `getAuthorizedApplications` to also resolve group/role assignments for user owners 🤔  but that can be handled with https://github.com/camunda/camunda/issues/26961

Reply:
This interface probably needs general refactoring as we need to take the ownerType into account anyway with https://github.com/camunda/camunda/issues/26961 as ids could collide

Reply:
🔧 For the scope we cover now, being user auths only I'm also fine not doing this change right now.